src/test/scala/scynamo/ScynamoCodecProps.scala

```diff
@@ -83,19 +83,24 @@ class ScynamoCodecProps extends Properties("ScynamoCodec") {
   propertyWithSeed("decode.encode === id (option)", propertySeed) = Prop.forAll { value: Option[Int] => decodeAfterEncodeIsIdentity(value) }
 
   propertyWithSeed("decode.encode === id (finite duration)", propertySeed) =
-    Prop.forAll(Gen.chooseNum[Long](-9223372036854775807L, 9223372036854775807L)) { value: Long =>
+    Prop.forAll(Gen.chooseNum(-9223372036854775807L, 9223372036854775807L)) { value =>
```

Comment:
This is just `Long.MinValue` and `Long.MaxValue`, so couldn't we just do this?
```suggestion
    Prop.forAll { value: Long =>
```

Reply:
It's not actually

Reply:
How weird is it that `Duration` supports from `Long.MinValue + 1` to `Long.MaxValue`? 😕 
Feel free to release a new version.